dropwizard-jersey/src/main/java/io/dropwizard/jersey/DropwizardResourceConfig.java

```diff
@@ -67,17 +67,20 @@ public DropwizardResourceConfig(@Nullable MetricRegistry metricRegistry) {
         register(loggingListener);
 
         register(new InstrumentedResourceMethodApplicationListener(metricRegistry));
-        register(CacheControlledResponseFeature.class);
-        register(io.dropwizard.jersey.guava.OptionalMessageBodyWriter.class);
-        register(new io.dropwizard.jersey.guava.OptionalParamBinder());
-        register(io.dropwizard.jersey.optional.OptionalMessageBodyWriter.class);
-        register(io.dropwizard.jersey.optional.OptionalDoubleMessageBodyWriter.class);
-        register(io.dropwizard.jersey.optional.OptionalIntMessageBodyWriter.class);
-        register(io.dropwizard.jersey.optional.OptionalLongMessageBodyWriter.class);
+        register(new CacheControlledResponseFeature());
+        register(new io.dropwizard.jersey.guava.OptionalMessageBodyWriter());
+        register(new io.dropwizard.jersey.optional.OptionalMessageBodyWriter());
+        register(new io.dropwizard.jersey.optional.OptionalDoubleMessageBodyWriter());
```

Comment:
Should we register these as instances or classes going forward? I thought part of the issue with the latest Jersey was that it no longer supported registering instances?